### DISCUSSION TITLE
PlanetScale as data provider

### DISCUSSION THREAD

**Discussion by onwp** [refinedev/refine#4098] [Integration request]
Hi peeps!
What do you think about PlanetScale? 

I think it would be a nice option for refine. 
Maybe together with Prisma. If not at least PlanetScale itself, since it's a serverless MySQL provider, it's a nice one. 

Thanks.

**Comment by BatuhanW** [comment]
Thanks for the suggestion! We think it's a great idea, would you like to help with the implementation?
